lib/active_shipping/carriers/fedex.rb

```diff
@@ -621,11 +621,12 @@ def parse_tracking_response(response, options)
 
           location = Location.new(:city => city, :state => state, :postal_code => zip_code, :country => country)
           description = event.at('EventDescription').text
+          type_code = event.at('EventType').text
 
           time          = Time.parse(event.at('Timestamp').text)
           zoneless_time = time.utc
 
-          shipment_events << ShipmentEvent.new(description, zoneless_time, location)
+          shipment_events << ShipmentEvent.new(description, zoneless_time, location, description, type_code)
```

Comment:
It was that, or pass `nil`. USPS opted to [add `description` twice](https://github.com/Shopify/active_shipping/blob/master/lib/active_shipping/carriers/usps.rb#L626) so I went with that.